ds3-cli-certification/src/main/java/com/spectralogic/ds3cli/certification/CertificationUtil.java

```
@@ -0,0 +1,91 @@
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3cli.certification;
+
+import com.spectralogic.ds3cli.Arguments;
+import com.spectralogic.ds3cli.CommandResponse;
+import com.spectralogic.ds3cli.Ds3ProviderImpl;
+import com.spectralogic.ds3cli.FileSystemProviderImpl;
+import com.spectralogic.ds3cli.command.CliCommand;
+import com.spectralogic.ds3cli.command.CliCommandFactory;
+import com.spectralogic.ds3cli.util.Ds3Provider;
+import com.spectralogic.ds3cli.util.FileSystemProvider;
+import com.spectralogic.ds3client.Ds3Client;
+import com.spectralogic.ds3client.commands.spectrads3.*;
+import com.spectralogic.ds3client.helpers.Ds3ClientHelpers;
+import com.spectralogic.ds3client.models.SpectraUser;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+
+public class CertificationUtil {
+
+    private static final ch.qos.logback.classic.Logger LOG =  (ch.qos.logback.classic.Logger) LoggerFactory.getLogger(CertificationUtil.class);
+
+    public static Path createTempFiles(
+           final String prefix,
+           final int numFiles,
+           final long length) throws IOException {
+       final Path tempDir = Files.createTempDirectory(prefix);
+       for(int fileNum = 0; fileNum < numFiles; fileNum++) {
+           final File tempFile = new File(tempDir.toString(), prefix + "_" + fileNum);
+           final RandomAccessFile raf = new RandomAccessFile(tempFile, "rw");
+           raf.seek(length);
+           raf.writeBytes("end of RandomAccessFile.");
+           raf.close();
+       }
+       return tempDir;
+    }
+
+    public static String getUserSecretKey(final Ds3Client client, final String username) {
+        try {
+            return client.getUserSpectraS3(new GetUserSpectraS3Request(username)).getSpectraUserResult().getSecretKey();
+        } catch (final Exception e) {
+            return "";
```

We should not swallow exceptions without at least logging the exception.